README.md

```diff
@@ -72,6 +72,7 @@ If you would like to get a head start with the tools we will be concentrating on
   * [Glue](http://glueviz.org/)
   * [SpecViz](http://specviz.readthedocs.io/en/latest/)
   * [Generalized WCS](http://gwcs.readthedocs.io/en/stable/)
+  * [Git and Github](https://guides.github.com/activities/hello-world/)
```

Comment:
Maybe also add the branching tutorial? Maybe it's just me, but I find it the most amazing hands-on resource.
```suggestion
  * [Git and Github](https://guides.github.com/activities/hello-world/)
  * [Git branching](https://learngitbranching.js.org/) 
```

Reply:
Sure, why not, I added it (sorry, didn't do it within GitHub, but within my branch).